pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -497,6 +497,10 @@ func (k *Kubernetes) VolumesFrom(objects *[]runtime.Object, komposeObject kobjec
 				if err != nil {
 					return errors.Wrap(err, "k.findDependentVolumes")
 				}
+				err = checkSimilarMount(volumeMounts, t.Spec.Template.Spec.Containers[0].VolumeMounts)
+				if err != nil {
+					log.Fatal(err)
```

Comment:
before every fatal out can we should mention which service has this problem? So it's easy where ther is a repetitive volume Mount?